### PR TITLE
adding aria-label to meter-radial to support a11y

### PR DESCRIPTION
**Pull request by jlai-d2l**

### CODE REVIEW

File: components/meter/meter-radial.js [hunk 1]
```diff
@@ -44,8 +44,12 @@ class MeterRadial extends LitElement {
 		const lengthOfLine = 115; // found by approximating half the perimeter of the ellipse with radii 38 and 35
 		const progressFill = this.value / this.max * lengthOfLine;
 		const secondaryTextElement = this.text ? html`<div class="d2l-body-small d2l-meter-radial-text">${this.text}</div>` : html``;
+		const label = this.text ? `${this.text}, progress indicator` : 'progress indicator';
```
Comment (m6jones) [inline components/meter/meter-radial.js]: We will want to always have primary text to show too. `${this.value} / ${this.max}` in the middle - `${this.text}, ${this.value} / ${this.max}, progress indicator`

File: components/meter/meter-radial.js [hunk 2]
```diff
@@ -44,8 +44,14 @@ class MeterRadial extends LitElement {
 		const lengthOfLine = 115; // found by approximating half the perimeter of the ellipse with radii 38 and 35
 		const progressFill = this.value / this.max * lengthOfLine;
 		const secondaryTextElement = this.text ? html`<div class="d2l-body-small d2l-meter-radial-text">${this.text}</div>` : html``;
+		let label = `${this.value} / ${this.max}, progress indicator`;
```
Comment (dbatiste) [inline components/meter/meter-radial.js]: This will need translation, yes?

Comment (jlai-d2l) [reply components/meter/meter-radial.js]: In speaking with @m6jones, this is a known gap. There currently any localization yet, and will need to be done. It probably makes sense to do this within a separate PR though.

Comment (dbatiste) [reply components/meter/meter-radial.js]: Ok sounds good.

**Pull request merged**